jiant/tasks/senteval_probing.py

```diff
@@ -0,0 +1,507 @@
+""" 
+Set of probing tasks that were added to Senteval Probing.
+Paper: https://arxiv.org/abs/1805.01070
+"""
+import collections
+import itertools
+import json
+import logging as log
+import os
+
+import numpy as np
+import pandas as pd
+import torch
+
+
+# Fields for instance processing
+from jiant.utils.data_loaders import tokenize_and_truncate
+from jiant.tasks.registry import register_task  # global task registry
+from jiant.tasks.tasks import SingleClassificationTask, process_single_pair_task_split
+
+
+@register_task("se-probing-sentence-length", rel_path="sentence_length/")
+class SEProbingSentenceLengthTask(SingleClassificationTask):
+    """ Sentence length task   """
+
+    def __init__(self, path, max_seq_len, name, **kw):
+        """ """
+        super(SEProbingSentenceLengthTask, self).__init__(name, n_classes=7, **kw)
+        self.path = path
+        self.max_seq_len = max_seq_len
+        self._label_namespace = self.name + "_tags"
+        self.train_data_text = None
+        self.val_data_text = None
+        self.test_data_text = None
+
+    def get_all_labels(self):
+        return [str(x) for x in list(range(6))]
+
+    def get_sentences(self):
+        return self.sentences
+
+    def load_data(self):
+        """ Load data """
+
+        def load_json(data_file):
+            rows = pd.read_csv(data_file, encoding="ISO-8859-1")
```

Comment:
random question: why do we need this encoding and not utf-8? Are there special characters in these tasks ?